### PR TITLE
Add Pull Request Template 

### PR DESCRIPTION
**Pull request by filipefilardi**
Create PR Template to make easier to contribute to this repo and easier to check if the pull request is consistent and followed all guidelines.

If this repo had my pull request, this commit would be like this:

##### Pull Request Checklist

- [x] Read [contributing guideline](CONTRIBUTING.md).
- [x] Read [code of conduct](CODE_OF_CONDUCT.md).
- [x] Fill [Contributor License Agreement (CLA)](https://cla.developers.google.com/).
- [x] Check if my changes are consistent with the [guidelines](https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#general-guidelines-and-philosophy-for-contribution).
- [x] Changes are consistent with the [Coding Style](https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#c-coding-style)
- [x] Run [Unit Tests](https://github.com/tensorflow/tensorflow/blob/master/CONTRIBUTING.md#running-unit-tests).

##### Issue Fix

- [ ] Yes
- [x] No

Fixed issue: none

##### Description

Add pull request template ...

### CODE REVIEW

**Comment by martinwicke** [comment]
I like this. @ewilderj, @yifeif, WDYT?

**Comment by filipefilardi** [comment]
Thank you for your reply @martinwicke.

I think having a PR template it's good to standardize PR and consequently save time of contributors and reviewers.

This one is just a first suggestion and can be improved if necessary.

**Comment by yifeif** [comment]
This is great. Thanks @filipefilardi!

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<!-- tensorflow pull request template -->
+
+##### Pull Request Checklist
+<!-- Before sending your pull requests, make sure you followed this list and tick all items -->
+- [ ] Read [contributing guideline](CONTRIBUTING.md).
```
Comment (ewilderj) [inline PULL_REQUEST_TEMPLATE.md]: s/guideline/guidelines/

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] Read [code of conduct](CODE_OF_CONDUCT.md).
```
Comment (ewilderj) [inline PULL_REQUEST_TEMPLATE.md]: please capitalize: Code of Conduct

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- [ ] Fill [Contributor License Agreement (CLA)](https://cla.developers.google.com/).
```
Comment (ewilderj) [inline PULL_REQUEST_TEMPLATE.md]: s/Fill/Ensure you have signed the/ ?

**Comment by filipefilardi** [comment]
Many thanks @ewilderj 

I made the changes as suggested and i do believe this might be better.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @martinwicke: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Review by martinwicke — APPROVED** [review]

**Pull request merged**